config/crds/serving_v1alpha1_knativeserving_crd.yaml

```diff
@@ -74,6 +74,16 @@ spec:
                   type: object
                   additionalProperties:
                     type: string
+            disabled-components:
```

Comment:
Maybe skip-install is a better name? disabled-components could imply disabling some serving-operator component vs knative-serving component.

Reply:
I'm open to `skip-install`, that works for me.